sonic-net/sonic-buildimage
# Include DPU module to sonic-chassis-module.yang

#### Why I did it
The modules have been extended to DPUx to support SmartSwitch.  As part of the SmartSwitch development module "DPUx" needs to be supported in every section of the code that referring to modules.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Updated "src/sonic-yang-models/yang-models/sonic-chassis-module.yang" file with "|DPU[0-9]" dpu module name.

#### How to verify it
1. Update "src/sonic-yang-models/tests/yang_model_tests/tests_config/chassis_module.json" to include DPUx modules and update /src/sonic-yang-models/tests/yang_model_tests/tests/chassis_module.json to verify DPU modules
2.  Update  "src/sonic-yang-models/tests/files/sample_config_db.json"  to include DPUx modules and verify using src/sonic-yang-mgmt/tests/libyang-python-tests/test_sonic_yang.py

#### Which release branch to backport (provide reason below if selected)
No need to brackport

#### Tested branch (Please provide the tested image version)
master

#### Description for the changelog
Added DPU modules to sonic-chassis-module.yang

#### Link to config_db schema for YANG module changes
Updated  https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/doc/Configuration.md

#### A picture of a cute animal (not mandatory but encouraged)
